### PR TITLE
upgrade php-http libs

### PR DESCRIPTION
**Pull request by kbond**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,16 +14,15 @@
     "require": {
         "symfony/framework-bundle": "~2.5|~3.0",
         "symfony/console": "~2.5|~3.0",
-        "php-http/adapter": "~0.1.0",
-        "php-http/message-factory": "~0.1.0",
-        "php-http/discovery": "^0.1.1"
+        "php-http/httplug": "~1.0@dev",
+        "php-http/message-factory": "~0.3",
```
Comment (sagikazarmark) [inline composer.json]: There is an 1.0 stable version now.

File: composer.json [hunk 2]
```diff
@@ -14,16 +14,15 @@
...
+        "php-http/discovery": "~0.2"
```
Comment (sagikazarmark) [inline composer.json]: The latest MJZ release is 0.6 (which uses [Puli](http://puli.io))

**Comment by sagikazarmark** [comment]
BTW not sure if you are aware that @dbu started to work on an HttplugBundle which provides a big part of the HTTP Client construction and configuration you implemented as well.

**Comment by kbond** [comment]
@sagikazarmark unfortunately I had to remove the discovery component as it relies on a beta version of puli/composer-plugin. When everything gets stabilized, I will re-add or perhaps just rely on @dbu's bundle.

**Comment by sagikazarmark** [comment]
@kbond we plan to remove that dependency for the long term:

php-http/discovery#49

**Pull request merged**